lib/pbench/test/unit/agent/task/test_results_move.py

```diff
@@ -91,6 +94,68 @@ def test_args():
             "Status: total "
         ), f"Unexpected stdout: '{result.stdout}', stderr: '{result.stderr}'"
 
+    @staticmethod
+    @responses.activate
+    def test_metadata_args():
+        """Test metadata with irregular option/value pair"""
+        runner = CliRunner(mix_stderr=False)
+        result = runner.invoke(
+            main,
+            args=[
+                TestMoveResults.CTRL_SWITCH,
+                TestMoveResults.CTRL_TEXT,
+                TestMoveResults.TOKN_SWITCH,
+                TestMoveResults.TOKN_TEXT,
+                TestMoveResults.DELY_SWITCH,
+                TestMoveResults.XZST_SWITCH,
+                TestMoveResults.SWSR_SWITCH,
+                TestMoveResults.SWSR_TEXT,
+                TestMoveResults.ACCESS_SWITCH,
+                TestMoveResults.ACCESS_TEXT,
+                TestMoveResults.META_SWITCH,
+                TestMoveResults.META_TEXT_TEST,
+                TestMoveResults.META_TEXT_FOO,
+            ],
+        )
+        assert (
+            result.exit_code == 2
+        ), f"Expected success exit code of 0: exit_code = {result.exit_code:d}, stderr: {result.stderr}, stdout: {result.stdout}"
+        assert result.stderr, f"Usage: '{result.stderr}', stdout: '{result.stdout}'"
```

Comment:
It seems like this assertion should be more specific than just, "Did we get something on `stderr`?"  And, the token "Usage" seems weird in this context -- it seems like the message should mention that `stderr` is unexpectedly empty.